examples/calling-apis/langchain-examples/src/agents/agent.py

```diff
@@ -46,9 +41,9 @@ def route_after_llm(state: State):
         ToolNode(
             [
                 # A tool with federated connection api access
-                with_calender_free_busy_access(check_user_calendar_tool),
-                # A tool without federated connection api access
-                check_country_holiday_tool,
+                check_user_calendar_tool,
+                list_github_repositories_tool,
+                # ... any other tool without federated connection api access
```

Comment:
if you are removing the wrapper function, it's not clear what tool has fed conn api access. Consider removing the comments

Reply:
the last comment?

Reply:
ok, consider removing both comments, they are useless IMO